### ISSUE TITLE
I can't seems to get anything to work

### ISSUE THREAD

**Issue by TheGreatBorg** [hardillb/node-red-node-geofence#11]
Please explain what to inject to get an output from this Geofence node. I have tried everything from JSON files with location sections inside but nothing get through this Geofence node. I expected something like add data to the msg.payload but no not a thing is added.

**Comment by hardillb** [comment]
The readme and the info sidebar are very clear on exactly which msg fields it works on.

Provide an example of what you've tried

**Comment by hardillb** [comment]
Also see https://github.com/hardillb/node-red-node-geofence/issues/10

**Comment by TheGreatBorg** [comment]
> The readme and the info sidebar are very clear on exactly which msg fields it works on.
> 
> Provide an example of what you've tried

Thank you very much for your rapid reply I really appreciate it.

I can OK read this from the description but what is it actually saying (maybe my imagination is not good enough to figure that out):

_This node requires input messages one of the following (checked in order):

- msg.location.lat & msg.location.lon
- msg.lat & msg.lon
- msg.payload.lat & msg.payload.lon

Alternatively it will add msg.location.inarea to the msg. with values of 
true/false_

If I read this description I can see that a msg has to be send to this note I suppose it must be in form of a JSON object (or not) and it has a data element with location and the 2 informations about lat and lon. I have made several attempts. How should I else make a string with these elements (I cannot figure that out) - help description could a bit more "helpful" I thing.. 

I have made this inject node for testing and send it directly to the GeoFence node and send it through a JSON node - nothing makes a difference:

![image](https://user-images.githubusercontent.com/31419036/55278241-fe6edc00-5309-11e9-86c3-acce5b113560.png)

And then I have made a GeoFence that should say that this location is outside the fence.

**Comment by hardillb** [comment]
The inject node sets the `msg.payload` value, not the whole `msg` object, so in your case msg object looks like the following:

    {
      payload: {
        name : "Test",
        location: {
          lat: 56.150466,
          lon: 9.999744
        }
      }
    }

Which places the location values at `msg.payload.location.lon` and `msg.payload.location.lat`

As you can see this does not match any of the locations checked by the node.

Make sure you set debug nodes to show the whole message and not just the `msg.payload` value.

If you want to use a inject node then you should set the payload value to:


    {
       name: "test",
       lat: 56.150466,
       lon: 9.99974
    }

**Comment by hardillb** [comment]
Also in future please use the node-red forum or Slack to ask general questions, issues are for actual problems with the code

**Comment by TheGreatBorg** [comment]
> Also in future please use the node-red forum or Slack to ask general questions, issues are for actual problems with the code

Sure I barely post issues on Github. But I searched a lot and found only this place regarding the GeoFence node. But I would have saved a lot of wasted time if that small example was showed in the node description (quiet knowledge makes easy entrance to the world of Node-red difficult). 

But the example you suggested is not accepted by an inject node (bad string) - so I still had to struggle a bit. But I found it and I can now start to figure out the logic behind this node. But I still think its an minor error that I get a false negative if the node-message is wrong formatted. I should get a message that msg. doesn't contain information for the GeoFence node to give a correct response. 

Here is my inject node I had to fix:

![image](https://user-images.githubusercontent.com/31419036/55282487-e2d4f700-5344-11e9-9764-ffeb191b9938.png)
